src/yul.rs

```diff
@@ -309,7 +334,8 @@ mod tests {
     fn literal() {
         assert_eq!(
             Literal {
-                literal: "testliteral".to_string()
+                literal: "testliteral".to_string(),
+                location: None,
```

Comment:
I did it the lazy way and just used empty locations in tests. It works because the thing that is being compared is the text representation and that does not have them.

I could add them but we'd also have to change the way they're tested. Any suggestions on how to do that without having just manually compare all the AST nodes one by one?

Reply:
We could have expectation files with the source locations of the nodes in some order